src/test/java/com/karl/ServerIT.java

```diff
@@ -19,13 +19,13 @@
   class when_server_is_started {
     private Client client;
 
-    @BeforeAll
```

Comment:
`ServerIT` == `ServerIntegrationTests`?

Reply:
yes, its the default convention for the testrunner being used, and i was too lazy to have it search for a new, more descriptive file blob.

Reply:
Yeah no worries, `ServerIT` works for me just wanted to make sure I understood.